pyproject.toml

```diff
@@ -71,15 +71,19 @@ module = [
 ]
 ignore_missing_imports = true
 
-[tool.uv]
-dev-dependencies = [
```

Comment:
`dev-dependencies` will be deprecated in the future.
Changed it to `dependency-groups`.
https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies